sixpack/sixpack.py

```diff
@@ -106,10 +106,12 @@ def get_response(self, endpoint=None, params=None):
         try:
             response = requests.get(url, params=params, timeout=self.timeout)
             if response.status_code != 200:
-                ret = "{{\"status\": \"failed\", \"response\": {0}}}".format(response.content)
+                try:
+                    content = response.json()
+                except Exception:
```

Comment:
I guess thats fair. My only other thought would be to log the error, but we don't seem to do that elsewhere so *shrug*.